### PR TITLE
:memo: Update README

### PR DESCRIPTION
**Pull request by shuklaritvik06**
Updated README file of Dev Protocol Organization

**Change Done**
Change

### CODE REVIEW

File: profile/README.md [hunk 1]
```diff
@@ -1,71 +1,158 @@
...
+<br>
+<br>
+<p><a href="http://devprotocol.xyz/" target="blank">Dev Protocol</a> is a solution for decentralized creator economy, it is designed for the blockchain solution and enhance creators to create their economy. We love <strong>$DEV</strong> and <strong>Stakes.social</strong>. Dev Protocol is not a single platform, it’s a middleware. Dev Protocol is completely decentralized and is a middleware protocol for on-chain governance, anyone can join the ecosystem by developing various Dapps. Dev Protocol is not closed to OSS, but can bring value to all open assets such as research papers, YouTube videos, and Spotify music. Dev Protocol integrates with these markets so that projects or creators can authenticate and tokenize their work. Developers and Sponsors both are rewarded with Tokens</p>
+<br>
+
+## Most Important Repositories
+
+#### Protocol core
```
Comment (Panquesito7) [inline profile/README.md:18]: Missing an extra line.
```suggestion
#### Protocol core

```

File: profile/README.md [hunk 2]
```diff
@@ -1,71 +1,158 @@
...
+   <!-- protocol-v2 -->
+   <td width=500px><h2 align="center"><a href="https://github.com/dev-protocol/protocol-v2">Protocol-v2</a></h2><p align="center"><a href="https://github.com/dev-protocol/protocol-v2/issues"><img src="https://img.shields.io/github/issues/dev-protocol/protocol-v2"></a>&nbsp;<a href="https://github.com/dev-protocol/protocol-v2/pulls"><img src="https://img.shields.io/github/issues-pr/dev-protocol/protocol-v2"></a></p><p align="center">🗞️ Smart contracts for Optimistic Rollups.</p></td>
+ </tr>
+  <tr></tr>
+      <tr>
+  <!-- s-tokens -->
+   <td width=500px><h2 align="center"><a href="https://github.com/dev-protocol/s-tokens">s-tokens</a></h2><p align="center"><a href="https://github.com/dev-protocol/s-tokens/issues"><img src="https://img.shields.io/github/issues/dev-protocol/s-tokens"></a>&nbsp;<a href="https://github.com/dev-protocol/s-tokens/pulls"><img src="https://img.shields.io/github/issues-pr/dev-protocol/s-tokens"></a></p><p align="center">💞 NFTs mirroring DEV staking
+.</p></td>
```
Comment (Panquesito7) [inline profile/README.md:30]:
```suggestion
   <td width=500px><h2 align="center"><a href="https://github.com/dev-protocol/s-tokens">s-tokens</a></h2><p align="center"><a href="https://github.com/dev-protocol/s-tokens/issues"><img src="https://img.shields.io/github/issues/dev-protocol/s-tokens"></a>&nbsp;<a href="https://github.com/dev-protocol/s-tokens/pulls"><img src="https://img.shields.io/github/issues-pr/dev-protocol/s-tokens"></a></p><p align="center">💞 NFTs mirroring DEV staking.</p></td>
```

**Review by Panquesito7 — APPROVED** [review]
LGTM 🚀 Thank you for your contribution! 😄👍

**Review by krishguptadev — APPROVED** [review]

**Review by vinzvinci — APPROVED** [review]
LGTM 👍

**Comment by vinzvinci** [comment]
CC @aggre

**Comment by aggre** [comment]
LGTM!

**Review by vinzvinci — APPROVED** [review]

**Pull request merged**